### PR TITLE
Add a "Future plans" section

### PR DESCRIPTION
**Pull request by certik**

### CODE REVIEW

**Review by gnikit — APPROVED** [review]
LGTM.

**Comment by gnikit** [comment]
One think I noticed, is that we don't have the roadmap for the LFortran LSP online yet. Let's comment out the:

```
Have a look at our roadmap (link) and consider contributing to our efforts (donations and contribution tasks link).
```

and uncomment it when we get around to having donation links, and a Roadmap/TODO list for that contributors can help with.

File: README.md [hunk 1]
```diff
@@ -78,6 +78,23 @@ The name of executable for this project has been chosen to remain `fortls`
...
+`fortls` has reached a point where it is feature complete and stable enough to be used in many modern Fortran projects without any issues.
+It does however still have fundamental limitations,
+namely its ability to understand all Fortran syntax and semantics that has been used throughout the 65+ years of the language. **The good news is that we have a plan to address this issue!**
+
+We are excited to announce that we are working on creating a new Fortran Language Server
+based on the actively developed [LFortran](https://lfortran.org/) compiler 🎉.
+The new Language Server will be able to understand all Fortran syntax, be faster,
+and give more accurate autocompletion, hover and diagnostic information. That means we plan on investing any future funding on creating our new language server and ultimately creating a better user experience for everyone.
+
+Have a look at our roadmap (link) and consider contributing to our efforts (donations and contribution tasks link).
```
Comment (gnikit) [inline README.md:92]:
```suggestion
<!--  Have a look at our roadmap (link) and consider contributing to our efforts (donations and contribution tasks link).  -->
```

**Comment by certik** [comment]
Perfect, thanks @gnikit !

**Pull request merged**